test-perl requires perl-parallel-forkmanager which is currently still in
testing, we therefore remove the test-perl prerequisites of the
install-bin target for now.

diff -upr cloc-1.76.orig/Unix/Makefile cloc-1.76/Unix/Makefile
--- cloc-1.76.orig/Unix/Makefile	2018-02-10 18:09:16.849581838 +0100
+++ cloc-1.76/Unix/Makefile	2018-02-10 18:09:24.872917920 +0100
@@ -105,7 +105,7 @@ install-man: test-pod man
 	$(INSTALL_BIN) -d $(MANDIR1)
 	$(INSTALL_DATA) $(INSTALL_OBJS_MAN) $(MANDIR1)
 
-install-bin: test-perl
+install-bin:
 	# install-bin - Install programs
 	$(INSTALL_BIN) -d $(BINDIR)
 	for f in $(INSTALL_OBJS_BIN); \
